qute/utilities/styling.py

```diff
@@ -97,12 +97,14 @@ def apply(styles, apply_to, **kwargs):
             continue
 
         # -- Add this extracted data to the compounded style
-        compounded_style += '\n' + extracted_style
+        compounded_style.append(extracted_style)
```

Comment:
str being immutable means str.__add__ makes a copy.
Usually better to str.join a long list than doing str.__add__ alot.